docs/using-airbyte/mappings.md

```diff
@@ -38,7 +40,7 @@ There are many reasons you might want to encrypt data.
 - **Efficient retrieval**: Hashing can enable faster lookups in databases.
 - **Anonymity and compliance**: Source datasets can contain personally identifiable information (PII). Anonymizing PII can help you meet data privacy regulations like GDPR and HIPAA.
 
-<!-- Probably need some guidance on the use of keys with Airbyte, esp. wrt external secrets managers and AES encryption -->
+<!-- Probably need some guidance on the use of keys with Airbyte, esp. wrt external secrets managers and AES encryption, and the validation rules -->
```

Comment:
@pedroslopez or I can probably answer whatever questions you have here.  Is this blocking?